docs/src/inflation.md

```diff
@@ -0,0 +1,45 @@
+# Inflation 
+Inflation is a method which systematically adds stochastic perturbations to the parameter ensemble in the form of gaussian noise. 
+Following the standard Kalman update, the parameter ensemble is perturbed in the following manner:
+```math
+    u_{n+1} = u_n + \zeta_{n} \qquad (1)
+```
+
+where `` \zeta_{n} `` is noise drawn from a gaussian distribution. Two distinct forms of inflation are implemented.
+For both implementations, a scaling factor ``s`` is included to extend usage to cases with mini-batching. For mini-batching,
+the scaling factor should be:
+```math
+    s = \frac{|B|}{|C|}
+```
+where `` |B| `` is the mini-batch size and `` |C| `` is the full dataset size, to account for sampling error. 
+
+## Multiplicative Inflation 
```

Comment:
mention that multiplicative inflation does not change the span of the ensemble, but additive inflation does (in their respective sections)